backend/src/packages/chaiNNer_ncnn/ncnn/processing/upscale_image.py

```diff
@@ -125,7 +133,7 @@ def estimate_cpu():
     schema_id="chainner:ncnn:upscale_image",
     name="Upscale Image",
     description="Upscale an image with NCNN. Unlike PyTorch, NCNN has GPU support on all devices, assuming your drivers support Vulkan. \
-            Select a manual number of tiles if you are having issues with the automatic mode.",
+            Select a manual number of tiles or set a memory budget limit if you are having issues with the automatic mode.",
```

Comment:
> or set a memory budget limit

1. This doesn't tell users how to do that.
2. This solution is factually incorrect. Setting a memory budget won't help at all in GPU mode.

Reply:
I'm not sure I follow; why would it not help in Vulkan mode?

Reply:
Sorry, my bad. I didn't see the `heap_budget` in `min(heap_budget, vkdev.get_heap_budget() * 1024 * 1024 * 0.8)`...

Point 1 still stands though.

Reply:
I don't see instructions on switching between GPU's in the backend metadata (though there is a mention of this in the README), which is why I didn't add similar instructions for this. Would you be favorable to adding a note in the backend docs metadata about making sure the integrated GPU isn't selected, along with a note about setting a memory budget? (Seems to me that documenting both of them in-app would be a UX improvement.)